src/components/Mint/MintPublic.tsx

```diff
@@ -60,6 +61,24 @@ const MintPublic: React.FC<Props> = ({
 			)
 			setLoadingMessage('Processing transaction ' + tx.hash)
 			await tx.wait()
+			ReactGA.event({
+				category: 'Minting',
+				action: 'Mint Public',
+				label: isNZ ? 'New Zealand' : 'Other',
+			})
+			ReactGA.plugin.execute('ec', 'setAction', 'purchase', {
+				transaction_id: tx.hash,
+				value: price.mul(quantity).toString(),
+				currency: 'eth',
+				affiliation: isNZ ? 'New Zealand Sale' : 'Other',
+				items: [
+					{
+						id: '1',
```

Comment:
What is this id? Can we use the peep server id here?

Reply:
Nope. This is for passport mint, at this point in the journey there is no peep token ID. I think technically `0` would be more accurate because its a passport token.

Reply:
Changed to `0`